extensions/sql-database-projects/package.json

```diff
@@ -165,10 +165,12 @@
     "menus": {
       "commandPalette": [
         {
-          "command": "sqlDatabaseProjects.new"
+          "command": "sqlDatabaseProjects.new",
```

Comment:
these 2 commands are no longer needed, right?

Reply:
will the data workspace create new project not call this eventually? I also just realized that SSDT projects aren't getting updated with the necessary .NET Core things when added in the new projects view(this used to happen in openProject() in the sql database extension). Created #12601 to track fixing this

Reply:
add project dialog will call the project provider api, not through command.

Reply:
ok I removed the commands for open, new, and close. Will clean up ProjectsController.openProject() and ProjectsController.newProject() later when the project provider api gets updated